docs/developer/guidelines/pr.md

```diff
@@ -30,18 +32,49 @@ section. For example:
 ...
 ```
 
-For changelog types, we adhere to those defined by [Keep a Changelog][keepachangelog-types]:
-
-- `Added` for new features or any non-trivial refactors.
-- `Changed` for changes in existing functionality.
-- `Deprecated` for soon-to-be removed features.
-- `Removed` for now removed features.
-- `Fixed` for any bug fixes.
-- `Security` in case of vulnerabilities.
-
-Every PR must add a changelog entry to each integration that has had its shipped code modified.
-
-The first line of every new changelog entry must end with a link to the PR in which the change
+Unless you have applied the `changelog/no-changelog` label, you must also edit the `CHANGELOG.md` file in 
+the integration you have modified. Below are the guidelines for the correct formatting for changelogs: 
+
+### Spacing
+
+* There should be one empty line between all text of different format. This means that there should be an 
+empty line between all of the following sections of text:
+  * Changelog file header
+  * Unreleased header
+  * Version / Date header
+  * Change type (ex: fixed, added, etc)
+  * Specific descriptions of changes (note: within this section itself, there should NOT be new lines between bullets)
+
+* `Extra spacing on line {line number}`: you have an extra empty line on the line referenced in the error. 
+There should be only one line of spacing!
+* `Missing spacing on line {line number}`: you need to add an empty line to the referenced lie
+
+### Integration Version Header
+
+* The header for an integration version should be in the following fashion: `version number / YYYY-MM-DD / Agent Version Number`. 
+The Agent version number is not necessary, but a valid version number and date are required. The first header after the 
+file's title can be `Unreleased`. The content under this section is the same as any other. 
+* `Version is formatted incorrectly on line {line number}`: The version you inputted is not a valid version, or there is 
+no / separator between the version and date in your header.
+* `Date is formatted incorrectly on line {line number}`: The date must be formatted as YYYY-MM-DD, with no spaces in between. 
+
+### Changelog Content
+* The changelog content is broken down by the different type of change that was made. For changelog types, we adhere 
+to those defined by [Keep a Changelog][keepachangelog-types]. For each different version, they must be written in the 
+following order, by priority:
+  * ***Removed***: for now removed features.
+  * ***Changed***: for changes in existing functionality.
+  * ***Security***: in case of vulnerabilities.
+  * ***Deprecated***: for soon-to-be removed features.
+  * ***Added***: for new features or any non-trivial refactors.
+  * ***Fixed***: for any bug fixes.
```

Comment:
```suggestion
  * ***Removed*** for features that have been removed.
  * ***Changed*** for changes to existing functionality.
  * ***Security*** in case of vulnerabilities.
  * ***Deprecated*** for soon-to-be removed features.
  * ***Added*** for new features or any non-trivial refactors.
  * ***Fixed*** for any bug fixes.
```